sunpy/net/helioviewer.py

```diff
@@ -168,34 +150,12 @@ def download_png(self, date, image_scale, layers, directory=None,
             0.6, 1.2, 2.4, and so on, increasing or decreasing by a factor
             of 2. The full-res scale of an AIA image is 0.6.
         layers : string
+            Image datasource layer/layers to include in the screeshot.
             Each layer string is comma-separated with these values, e.g.:
             "[sourceId,visible,opacity]" or "[obs,inst,det,meas,visible,opacity]"
             Multiple layer string are by commas: "[layer1],[layer2],[layer3]"
-        directory : string
-            (Optional)  Directory to download JPEG 2000 image to.
-        x1 : float
-            (Optional) The offset of the image's left boundary from the center
-            of the sun, in arcseconds.
-        y1 : float
-            (Optional) The offset of the image's top boundary from the center
-            of the sun, in arcseconds.
-        x2 : float
-            (Optional) The offset of the image's right boundary from the
-            center of the sun, in arcseconds.
-        y2 : float
-            (Optional) The offset of the image's bottom boundary from the
-            center of the sun, in arcseconds.
-        x0 : float
-            (Optional) The horizontal offset from the center of the Sun.
-        y0 : float
-            (Optional) The vertical offset from the center of the Sun.
-        width : int
-            (Optional) Width of the image in pixels (Maximum: 1920).
-        height : int
-            (Optional) Height of the image in pixels (Maximum: 1200).
-        watermark
-            (Optional) Whether or not the include the timestamps and the
-            Helioviewer.org logo in the image (Default=True).
```

Comment:
Do none of these work?

Do we still not want to support these?

Reply:
We have support for them but if we are not using **kwargs we cannot pass any of those neither do we need them as such because they just define some minor features of the image that will be downloaded. The image can also be downloaded without those optional parameters

Reply:
Ok I will add them